src/base/solvers/solveCobraLP.m

```diff
@@ -1228,13 +1228,59 @@
             ILOGcplex.solve();
 
             origStat   = ILOGcplex.Solution.status;
+            stat = origStat;
             if origStat==1
                 f = osense*ILOGcplex.Solution.objval;
                 x = ILOGcplex.Solution.x;
                 w = ILOGcplex.Solution.reducedcost;
-                y = ILOGcplex.Solution.dual;
+                y = ILOGcplex.Solution.dual;                                            
+            elseif origStat == 4
+                %This is likely unbounded, but could be infeasible 
+                %Lets check, by solving an additional LP with a bounded
+                %objective.
+                %Store the original solution
+                Solution = ILOGcplex.Solution;
```

Comment:
To be honest:
This is not an option, or rather, we would than have to add another option to do this or not to do this.
The problem arises in small networks, where the pre-solver already determines, that this is unbounded/infeasible, which CPLEX doesn't distinguish. So we actually never "solved" the problem, but only ran a presolver on it.
If we give a warning, than e.g. fva can't distinguish whether its unbounded or infeasible, and we would either have to test this again there and handle different options at all code places, where those make a difference. 
This return value is also very rare for any complex problem, so its not a computational time problem (in my opinion).